### ISSUE TITLE
Making displaying Wikipedia articles optional or change screen part size

### ISSUE THREAD

**Issue by Machion** [organicmaps/organicmaps#9051]
**Is your feature request related to a problem? Please describe.**
When I tab on a point on the map and this point has a Wikipedia article, the article gets displayed. 
The problem is, the article section covers much of the screen. Only a small part of the map stays visible to scroll/zoom and when I try to lower the arcticle section, it disappears immediately and my marked point gets unselected. I don't use these articles very often anyway. 

**Describe the ideal solution**
Maybe there should be more steps to shrink the article section or disable it completely without losing the marked point. 

**Describe alternatives you have considered**

**Additional context**

**Comment by biodranik** [comment]
@Machion is it iOS or Android? Can you please share a screenshot or a short screen video recording?

**Comment by Machion** [comment]
It's on Android. 
I have a resolution of 2520x1080. The screenshot below shows a division around 50% (for big articles). I could scroll up even more to reveal stuff like coordinates. 

![OM-screenshot](https://github.com/user-attachments/assets/7eda822b-9002-4764-b2e6-fc62809602d5)

**Comment by hoizan** [comment]
Yes, it should allow to close the Wikipedia page only, and the rest of the action buttons at the bottom of the map screen should stay.
